<launch>

  	<node pkg="map_compression" type="saliencyMap" name="saliencyMap" output="screen" >

	<param name="normalRadius" value="0.5"/>	
	<param name="fpfhRadius" value="1.0"/>
    <param name="accKSearch" value="100"/>	

    <param name="highRatio_pointAssoc" value="0.3"/>	
    <param name="sigma" value="0.5"/>	

    <param name="highRatio_highDistinct" value="0.3"/>	
    <param name="highRadius" value="2.0"/>
    <param name="highSearchNum" value="100"/>

<<<<<<< HEAD
	<param name="loadMapDir" value="/home/yinhuan/Data/mapModel/saliency/split_cloud/"/>
	<param name="subMapCnt" value="64"/>
  	<param name="saveSaliencyDir" value="/home/yinhuan/Data/mapModel/saliency/split_saliency/"/>
=======
	<!--param name="loadMapDir" value="/home/yh/mapModel/2019/06.27/test/"/>
	<param name="subMapCnt" value="1"/>
  	<param name="saveSaliencyDir" value="/home/yh/mapModel/2019/06.27/test/"/-->

	<param name="loadMapDir" value="/home/yh/mapModel/2019/07.11/compare/split_cloud/"/>
	<param name="subMapCnt" value="13"/>
  	<param name="saveSaliencyDir" value="/home/yh/mapModel/2019/07.11/compare/split_cloud/"/>
>>>>>>> c77e0212e5b1a1049c8fd58103c53b13eaff0256

  </node>
</launch>
